zcash_client_sqlite/src/wallet.rs

```diff
@@ -303,6 +314,27 @@ pub fn get_received_memo<P>(wdb: &WalletDb<P>, id_note: i64) -> Result<Memo, Sql
         .map_err(SqliteClientError::from)
 }
 
+pub fn get_transaction<P: Parameters>(
+    wdb: &WalletDb<P>,
+    id_tx: i64,
+) -> Result<Transaction, SqliteClientError> {
+    let (tx_bytes, block_height): (Vec<_>, BlockHeight) = wdb.conn.query_row(
+        "SELECT raw, block FROM transactions
+        WHERE id_tx = ?",
```

Comment:
Should this be a prepared statement?

Reply:
In this case I don't think it makes any real difference.